include/sd_sector_addresses.h

```diff
@@ -22,122 +22,251 @@ struct SectorAddress
+/**********
+ * Escape menu
+ **********/
+/**********
+ * Walk menu + home
+ **********/
+#define WALK_HOME SectorAddress { 0x0000, 0x213A }
+#define WALK_HOME_SELECTED SectorAddress { 0x0000, 0x218B }
+#define WALK_HOME_ACTIVATED SectorAddress { 0x0000, 0x21DC }
 
-// Side step left
-#define SIDE_STEP_LEFT_SMALL SectorAddress { 0x0000, 0x2F76 }
-#define SIDE_STEP_LEFT_SMALL_SELECTED SectorAddress { 0x0000, 0x3018 }
-#define SIDE_STEP_LEFT_SMALL_ACTIVATED SectorAddress { 0x0000, 0x2FC7 }
+#define HOME_WALKMENU SectorAddress { 0x0000, 0x222D }
+#define HOME_WALKMENU_SELECTED SectorAddress { 0x0000, 0x227E }
+#define HOME_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x22CF }
 
-#define SIDE_STEP_LEFT_NORMAL SectorAddress { 0x0000, 0x2C9D }
-#define SIDE_STEP_LEFT_NORMAL_SELECTED SectorAddress { 0x0000, 0x2D3F }
-#define SIDE_STEP_LEFT_NORMAL_ACTIVATED SectorAddress { 0x0000, 0x2CEE }
+/**********
+ * Single step menu + home
+ **********/
+#define SINGLE_STEP_HOME SectorAddress { 0x0000, 0x1F54 }
+#define SINGLE_STEP_HOME_SELECTED SectorAddress { 0x0000, 0x1FA5 }
+#define SINGLE_STEP_ACTIVATED SectorAddress { 0x0000, 0x1FF6 }
 
-// Side step right
-#define SIDE_STEP_RIGHT_SMALL SectorAddress { 0x0000, 0x3069 }
-#define SIDE_STEP_RIGHT_SMALL_SELECTED SectorAddress { 0x0000, 0x310B }
-#define SIDE_STEP_RIGHT_SMALL_ACTIVATED SectorAddress { 0x0000, 0x30BA }
+#define HOME_SSMENU SectorAddress { 0x0000, 0x2047 }
+#define HOME_SSMENU_SELECTED SectorAddress { 0x0000, 0x2098 }
+#define HOME_SSMENU_ACTIVATED SectorAddress { 0x0000, 0x20E9 }
 
-#define SIDE_STEP_RIGHT_NORMAL SectorAddress { 0x0000, 0x2D90 }
-#define SIDE_STEP_RIGHT_NORMAL_SELECTED SectorAddress { 0x0000, 0x2E32 }
-#define SIDE_STEP_RIGHT_NORMAL_ACTIVATED SectorAddress { 0x0000, 0x0561 }
+/**********
+ * Walk menu + home stand + sit + home
+ **********/
+#define WALK_4 SectorAddress { 0x0000, 0x31FF }
+#define WALK_4_SELECTED SectorAddress { 0x0000, 0x171A }
+#define WALK_4_ACTIVATED SectorAddress { 0x0000, 0x176B }
 
-// Single steps
-#define SINGLE_STEP_SMALL SectorAddress { 0x0000, 0x097E }
-#define SINGLE_STEP_SMALL_SELECTED SectorAddress { 0x0000, 0x0A20 }
-#define SINGLE_STEP_SMALL_ACTIVATED SectorAddress { 0x0000, 0x09CF }
+#define HOMESTAND_4_WALKMENU SectorAddress { 0x0000, 0x3250 }
+#define HOMESTAND_4_WALKMENU_SELECTED SectorAddress { 0x0000, 0x180D }
+#define HOMESTAND_4_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x185E }
 
-#define SINGLE_STEP_NORMAL SectorAddress { 0x0000, 0x0747 }
-#define SINGLE_STEP_NORMAL_SELECTED SectorAddress { 0x0000, 0x07E9 }
-#define SINGLE_STEP_NORMAL_ACTIVATED SectorAddress { 0x0000, 0x0798 }
+#define SIT_4_WALKMENU SectorAddress { 0x0000, 0x315D }
+#define SIT_4_WALKMENU_SELECTED SectorAddress { 0x0000, 0x1900 }
+#define SIT_4_WALKMENU_ACTIVATED SectorAddress { 0x0000, 0x1951 }
 
-#define SINGLE_STEP_LARGE SectorAddress { 0x0000, 0x0654 }
-/***** Walk menu *****/
+#define HOME_WAlKMENU SectorAddress { 0x0000, 0x31AE }
```

Comment:
This one already exists on line 77, but with different values
```suggestion
#define HOME_WALKMENU SectorAddress { 0x0000, 0x31AE }
```